pyscf/adc/radc_ao2mo.py

```diff
@@ -16,6 +16,7 @@
 #         Alexander Sokolov <alexander.y.sokolov@gmail.com>
 #
 
+import os
```

Comment:
Why do we need os? I don't see it being used.

Reply:
Removed os